src/meta/test/JobManagerTest.cpp

```diff
@@ -328,16 +356,48 @@ TEST_F(JobManagerTest, showJob) {
   ASSERT_EQ(tasks[1].get_stop_time(), td2.stopTime_);
 }
 
+TEST_F(JobManagerTest, showJobInOtherSpace) {
+  std::vector<std::string> paras{"test_space"};
+
+  JobDescription jd(1, cpp2::AdminCmd::COMPACT, paras);
+  jd.setStatus(cpp2::JobStatus::RUNNING);
```

Comment:
Redundant code

Reply:
Setting Jobs to RUNNING will record start time, and to phase after RUNNING will record stop time... while i don't check this, i want it to be right...

Reply:
ok